client/src/theme.js

```diff
@@ -125,6 +125,15 @@ const theme = createTheme({
         },
       ],
     },
+    MuiCard: {
+      styleOverrides: {
+        root: {
+          padding: "2rem",
+          borderRadius: "2rem",
+          boxShadow: "0px 3px 3px 0px #00000033",
```

Comment:
Did the card's default styling not apply the shadow/elevation we needed? (MUI uses the concept of elevation for different box shadow styles)

Reply:
oh i wasn't aware it had that prop

Reply:
although when i use the variant, the shadow/elevation is removed 
i'm thinking of just applying sx to the card and add the padding and borderradius there instead (and use elevation prop for the boxshadow)
the reason why i'm thinking that it might not be needed in theme because in the other cases that might use Cards (such as in the tracker), the padding and border radius are all different (the only thing alike for everything is the background colour, which is white as default)

Reply:
cool, if thats the case, i say we put as much of the common styles into the base MuiCard override, becuase creating variants that are only used in one place is just like creating a unique css class and having one off styles anyways.

So you could set up to just use `<Card sx={{..any other specifics}} />` if we're only using this for the internship card thats fine then.

might be some future iteration where branding is more important, we'll thank ourselves today if we have very little variations we've created and little to no hard coded styles. 

the dream is to just update the theme and it trickles down to each component as we expect

Reply:
wait i just realized that the sign up/login and account settings has a similar wrapper 

I wouldn't say these are cards though...
![Screen Shot 2023-05-12 at 6 44 31 PM](https://github.com/BobaTalks/internship-tracker/assets/96795538/889872d4-0f65-4
![Screen Shot 2023-05-12 at 6 44 42 PM](https://github.com/BobaTalks/internship-tracker/assets/96795538/df96be74-41dc-4ddd-bc75-619b5ef7c2ab)
b2a-8541-9417ea3b932c)

Also the reason why I used variants is because of these job tickets here which I would consider as cards but have different css properties

![Screen Shot 2023-05-12 at 6 49 56 PM](https://github.com/BobaTalks/internship-tracker/assets/96795538/28760c15-428d-4ae3-a350-cd4db105e940)

Would it be possible to have signup/login and account settings be cards as well (so we can apply the css to the root theme for MuiCards) and just create the job tickets as regular Box (it doesn't need any css or funcionality that Card provides like elevation or anything else). 

Or we can also utilize Paper instead for the internship component/signup/account settings and use Card for job tickets.

That being said, in general, should we use the MUI component that best describes what the element is or the MUI component that provides the props/functionality that will be the useful?

Reply:
yeah if we're looking at composition

signup might look something like
```jsx
<Card>
  <Form>
  <Button>
  <Divider>
  <Social Stuff>
  <Footer>
</Card>
```
the little internship cards might be

```jsx
<Card>
  <Logo and Name>
  <Details>
  <Date>
</Card>
```

The similarities look like the background, the border radius, maybe some responsive padding, and if those are the only styles we know are the same (or want to be the same) that very much warrants adding in an override to the MuiCard i think. Cause then the only time we really need to add styles, would be idk maybe in the tracker later, it looks like they don't have shadows on the small ones. BUT that doesn't mean it shouldnt, that would be a great question for design

Reply:
But yeah, card or paper, depending what we want to do. like "card" and "paper" i feel is kinda conceptual and legit, we just use card for everything at work lol, i shoudl take some time to read up on when Paper should be used over Card... 

But as long as we are consistent, things make sense, and everyone is in agreement in what they are used for, refactoring/changing it later should be minimal effort

Reply:
i've changed it! (I've also utilized elevation as you mentioned)